ibm/resource_ibm_is_vpn_gateway_connections.go

```diff
@@ -142,6 +148,57 @@ func resourceIBMISVPNGatewayConnection() *schema.Resource {
 				Computed:    true,
 				Description: "The crn of the VPN Gateway resource",
 			},
+
+			isVPNGatewayConnectionAdminAuthenticationmode: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Optional:    true,
+				Description: "The authentication mode",
+			},
+
+			isVPNGatewayConnectionResourcetype: {
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
isVPNGatewayConnectionResourcetype, isVPNGatewayConnectionCreatedat, isVPNGatewayConnectionMode should be only computed